lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,10 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
 <div class="recent-updates">
   % for index, update in enumerate(visible_updates):
     <article class="updates-article">
       % if not update.get("is_error"):
-        <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
+        <h5 class="date" id="msg-date-${index}">${update.get("date")}</h5>
```

Comment:
I wonder if this should just be a div. It doesn't really seem like a heading, and ``h5``s are semantically odd.